open-api-spec.yaml

```diff
@@ -813,6 +831,26 @@ components:
           assetCode: USD
           assetScale: 2
           authServer: 'https://openpayments.guide/auth'
+    json-web-key-set:
```

Comment:
Since this only includes keys and not client info, is the client (display) info expected to be found by querying the payment pointer itself?
Or is client display info something that the client provides in grant requests that the account provider doesn't attest to?
https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-2.3.2
For comparison, the current `/keys/{id}` endpoint in the Rafiki backend returns:
https://github.com/interledger/rafiki/blob/464ef810a8a47c718c4132a426c169c95d7ae8d7/packages/backend/src/clientKeys/routes.ts#L36-L45

Reply:
I think that pulling this data from the account provider is more reliable and opens us up to supporting use cases in future where this data can be verified (e.g. using DIDs and verifiable credentials)

Reply:
To be more specific, I think we should say that we only allow identifying the client by reference

Reply:
So how does the RS pull client display info from the account provider?

Reply:
What if we put this in the payment pointer response?

If you are a consumer that wants to protect your privacy then you wouldn't provide this but if you're a business and likely to act as a client then you would.

Reply:
Agree, but that won't be every time right?

Reply:
Yeah, it seems like the payment pointer (or AS grant) request would only happen once per grant?
Whereas the `jwks.json` request would be happening on every grant and token introspection request (unless the `auth` package starts caching keys).
It looks like the rafiki `auth` package is currently only storing `kid`. It seems like it should also store the public key (to bind to the grant)?
https://github.com/interledger/rafiki/blob/f17fdb9981d57d6199062a94ef032d47339b5459/packages/auth/migrations/20220504161932_create_grants_table.js#L15

Reply:
> the jwks.json request would be happening on every grant and token introspection request (unless the auth package starts caching keys)

I'd definitely suggest caching keys. We probably need to give some thought to an appropriate cache period.

Reply:
I think the request to the payment pointer to retrieve client display info will only need to happen on grant requests with `interact` specified.